ui/guest/src/react/ExperienceBuilder.tsx

```diff
@@ -112,7 +111,7 @@ import useUnmount from '@craftercms/studio-ui/hooks/useUnmount';
 interface BaseXBProps {
   documentDomain?: string;
   themeOptions?: ThemeOptions;
-  sxOverrides?: DeepPartial<GuestStylesSx>;
+  sxOverrides?: Partial<GuestStylesSx>;
```

Comment:
Did they remove `DeepPartial`? This might be problematic and perhaps we should replace it with our own.

**You must create scenarios to _throughly_ test all typing changes**